msgfileparser_connector.py

```diff
@@ -567,7 +452,7 @@ def _create_container(self, action_result, email_artifact, label, container_seve
         if not container.get('name'):
             container['name'] = os.path.basename(vault_path)
 
-        # we don't/can't add a raw_email to this container, .msg file does not contain the email
+        # we don't/can't add a raw_email to this container, .message file does not contain the email
```

Comment:
This one seems like it should stay `.msg` as a file extension

Reply:
good catch, fixed